consoles/sshVirtsh.pm

```diff
@@ -357,16 +357,18 @@ sub add_disk {
             # Avoid qemu-img's failure to get a write lock to be the reason for a job to fail
             while (1) {
                 my ($ret, $stdout, $stderr) = $self->run_cmd("qemu-img create $file $size -f qcow2", wantarray => 1);
-                die "Too many attempts to format HDD" unless ($bucket-- && $ret);
                 if ($stderr =~ /lock/i) {
+                    $bucket--;
+                    die "Too many attempts to format HDD" unless ($bucket);    #Only die when there are no attempts left
```

Comment:
that comment is a little too obvious and we don't use () if it's a simple statement

Reply:
@coolo updated